packages/camlpdf/camlpdf.2.3/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "2.0"
+maintainer: "contact@coherentgraphics.co.uk"
+authors: ["John Whitington"]
+homepage: "http://github.com/johnwhitington/camlpdf"
+bug-reports: "http://github.com/johnwhitington/camlpdf/issues"
+dev-repo: "git://github.com/johnwhitington/camlpdf"
+build: [[make]]
+install: [[make "install"]]
+remove: [["ocamlfind" "remove" "camlpdf"]]
+depends: ["ocaml" "ocamlfind"]
```

Comment:
```suggestion
depends: [
  "ocaml"
  "ocamlfind" {build}
]
```
`ocamlfind` should be tagged as a build dependency since this packages doesn't require to be rebuilt whenever ocamlfind is upgraded.